### DISCUSSION TITLE
Spring Tomcat application construction failes after java buildpack updates paketo-buildpacks/spring-boot to v5.29.0

### DISCUSSION THREAD

**Discussion by ksantosejie** [paketo-buildpacks/feedback#282] [General]
I'm a newcomer to this Paketo world. We have a Spring MVC application that generates a .war file, which is then deployed on a Tomcat. All of this was working fine until the Java Paketo buildpack upgrades and spring boot Paketo buildpack version was updated to v5.29.0. Since that update, we always get the following message: "**unable to find Spring Boot Executable Jar**" Our application is not a Spring Boot application, so indeed, there is no .jar file.

Analyzing the updated code, we see that the error comes from this check in the build.go file.

![imagen](https://github.com/paketo-buildpacks/feedback/assets/152895232/c004018a-43e2-4952-b788-24ceb1a276bf)

In previous versions, it was done in a different way.

![imagen](https://github.com/paketo-buildpacks/feedback/assets/152895232/4f1ed751-a230-4337-bb36-98076ec3e1c5)

So, if there wasn't a .jar file, as was our case, the Spring Boot Paketo did nothing and that was fine. Now, this generates an error.

We don't know if there's a way to bypass this validation or indicate that our application is not a Spring Boot application, so this part of the code doesn't execute.

Any help would be appreciated, as you can see, we're beginners in all of this.

**Comment by dmikusa** [comment]
Thanks for raising this up. Looking into it. Hope to have something more soon. You can follow https://github.com/orgs/paketo-buildpacks/discussions/282 for more details, but we'll try to update this thread as well.

**Reply by anthonydahanne** [reply]
@dmikusa , you meant https://github.com/paketo-buildpacks/spring-boot/issues/474

**Reply by dmikusa** [reply]
🤦  Indeed. Thanks.

**Reply by ksantosejie** [reply]
Thank you very much for the response. I'll be waiting then.

**Reply by pivotal-david-osullivan** [reply]
@ksantosejie Can you please try the latest release 13.0.1 and let us know if the issue is resolved?

**Reply by ksantosejie** [reply]
We confirm that now works as expected! Great work, thank you!
